goicicle/setup.sh

```diff
@@ -1,10 +1,24 @@
 #!/bin/bash
 
+SUDO=''
+if [ "$EUID" -ne 0 ]; then 
+  echo "Icicle setup script should be run with root priviledges, please run this as root"
```

Comment:
the message suggests that the script should be run as root, but it doesn't exit after displaying the message. This might be confusing for the user.

Reply:
RIght, it asks for root credentials before make